esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -701,6 +702,27 @@ bool EspHttpBinding::basicAuth(IEspContext* ctx)
         return false;
     }
 
+    //Check if the password is a "real" password, or a workunit distributed access token
+    const char * pwd = user->credentials().getPassword();
+    if (isWorkunitDAToken(pwd))
+    {
+        wuTokenStates state = verifyWorkunitDAToken(pwd);//throws if cannot open workunit
+        if (state == wuTokenValid)
+        {
+            user->setAuthenticateStatus(AS_AUTHENTICATED);
+        }
```

Comment:
Should this also clear the password?  Where is the password cleared if the user is authenticated?